### PR TITLE
[bitnami/common]: Add capabilities macros to manage Pod Security Standard objects

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

As discussed at https://github.com/bitnami/charts/issues/15355, we need to ensure the deprecated PodSecurityPolicy objects aren't used on K8s > 1.25 and pod security standards are use instead via the built-in admission controller.. Find more information in the links below:

- https://kubernetes.io/docs/concepts/security/pod-security-admission

This PR is a 1st step on this direction since we need macros to help us distinguishing based on the K8s version whether these objects are supported or not or choosing the appropriate apiVersion.

### Benefits

Pod Security Standards customization.

### Possible drawbacks

None

### Applicable issues

Related with https://github.com/bitnami/charts/issues/15355 

### Additional information

N/A

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by migruiz4 — COMMENTED** [review]
Looks good to me, but version 2.12.0 has already been released.
Please bump a new version so I can approve it.

**Review by migruiz4 — APPROVED** [review]

**Comment by juan131** [comment]
Sorry @migruiz4 had to merge `main` again since another PR affecting bitnami/common was merged

**Review by migruiz4 — APPROVED** [review]

File: bitnami/common/templates/_capabilities.tpl [hunk 1]
```diff
@@ -172,6 +172,50 @@ Return the appropriate apiVersion for Vertical Pod Autoscaler.
...
+{{- define "common.capabilities.admisionConfiguration.supported" -}}
+{{- if semverCompare ">=1.23-0" (include "common.capabilities.kubeVersion" .) -}}
+  {{- true -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the appropriate apiVersion for AdmissionConfiguration.
+*/}}
+{{- define "common.capabilities.admisionConfiguration.apiVersion" -}}
```
Comment (odessosivan) [inline bitnami/common/templates/_capabilities.tpl:196]: @juan131 I think I've spotted a typo and the word should be admission :slightly_smiling_face:

```suggestion
{{/*
Return the appropriate apiVersion for AdmissionConfiguration.
*/}}
{{- define "common.capabilities.admissionConfiguration.apiVersion" -}}
```

Comment (juan131) [reply bitnami/common/templates/_capabilities.tpl:196]: Good catch! Would you send a PR fixing it?

Comment (odessosivan) [reply bitnami/common/templates/_capabilities.tpl:196]: Raised https://github.com/bitnami/charts/pull/19840 as draft in case I've messed something up.

**Pull request merged**